Automattic/wp-calypso
# Add/onboarding wordpress importer

#### Changes proposed in this Pull Request

* Registered a WordPress onboarding flow `/start/from/importing/wordpress`
* Added feature flag `gutenboarding/import-from-wordpress`
* Added a *static* content type selector screen

#### Testing instructions

* go to `/start/from/importing/wordpress?from={WP_SITE_URL}&to={slug}.wordpress.com`
* check if there is a static "content type selector" screen

#### Screenshots

**Desktop**
<img width="1040" alt="Screenshot 2021-12-23 at 14 23 54" src="https://user-images.githubusercontent.com/1241413/147246744-b22b62f8-af6a-481a-bafe-7662a61daf79.png">

**Mobile**
<img width="377" alt="Screenshot 2021-12-23 at 14 34 51" src="https://user-images.githubusercontent.com/1241413/147247917-831341b9-55fe-4337-892b-fbb0eae7b361.png">

Related to #59042
